cmake/projects/OpenSSL/externals/nasm_path_configure.cmake.in

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Minimum for Hunter is 3.2:
* https://docs.hunter.sh/en/latest/quick-start/cmake.html

Reply:
Please fix this.